.gitignore

```diff
@@ -1,5 +1,4 @@
 build/*.o
```

Comment:
`build/*` would be enough, but that'll work just fine
Just don't create Yet Another Pull Request for that :)